board/views.py

```diff
@@ -41,7 +67,234 @@ def get(self, request):
+                comment_data = [{
+                    "id"        : comment['id'],
+                    "writer"    : Member.objects.get(id=comment['writer_id']).nickname,
+                    "content"   : comment['content'],
+                    "is_liked"  : CommentLike.objects.filter(comment_id = comment['id'], is_like = 1).count(), 
+                    "created_at": comment['created_at'],
+                    "if_i_liked": check_commentLike(self, request, comment['id']),
+                } for comment in data]
+            else:
+                comment_data = ''
+            return JsonResponse({'message' : 'SUCCESS', 'comment_data' : comment_data, 'page' : page}, status = 200)
+        except KeyError:
+            return JsonResponse({'message' : 'KEY_ERROR'}, status = 400)
+        except ValueError:
+            return JsonResponse({'message' : 'DECODE_ERROR'}, status = 400)
+        except Comment.DoesNotExist:
+            return JsonResponse({'message' : 'NO_EXIST_DATA'}, status = 500)        
+        except :
+            return JsonResponse({'message' : 'NO_COMMENT_EXIST'}, status = 500)
+
+    ## 2. 댓글 작성(POST)
+    @login_check
+    def post(self, request):
+        try:
+            if request.user != None:
+                member_id  = request.user.id
+            else:
+                return JsonResponse({"message" : "INVALID_USER"}, status=400) ## 확인부탁드려요!!
+
+            data        = json.loads(request.body)
+            board_id    = data['board_id']
+            content     = data['content']
+
+            if content == "":
+                return JsonResponse({'message' : 'COMMENT_REQUIRED'}, status=400)
+            else:
+                Comment.objects.create(
+                    content         = content,
+                    created_at      = datetime.datetime.now(),
+                    board_id        = board_id,
+                    writer_id       = member_id
+                )
+            return JsonResponse({'message' : 'SUCCESS'}, status = 201)
+        except KeyError:
+            return JsonResponse({'message' : 'KEY_ERROR'}, status = 400)          
+        except :
+            return JsonResponse({'message' : 'NO_COMMENT_EXIST'}, status = 500) 
+
+
+# 3. 대댓글 작성
+class AddSelfCommentView(View):
+    @login_check
+    def post(self, request):
+        try:
+            if request.user != None:
+                member_id  = request.user.id
+            else:
+                return JsonResponse({"message" : "INVALID_USER"}, status=400) ## 확인부탁드려요!!
+
+            data        = json.loads(request.body)
+            board_id    = data['board_id']
+            comment_id  = data['comment_id']
+            content     = data['content']
+
+            if board_id and member_id:
+                if content == "":
+                    return JsonResponse({'message' : 'COMMENT_REQUIRED'}, status=400)
+                elif Comment.objects.filter(id=comment_id).exists():
```

Comment:
```suggestion
                if Comment.objects.filter(id=comment_id).exists():
```
위에 내용검증 content 아래로 옮기면 여긴 이렇게 바꾸면 되겠죠.